packages/venia-ui/lib/components/MiniCart/product.js

```diff
@@ -39,27 +37,27 @@ const Product = props => {
     } = talonProps;
 
     const classes = mergeClasses(defaultClasses, props.classes);
+    const { image } = item;
 
     const productImage = useMemo(() => {
-        const src = hasImage
-            ? resourceUrl(image.url, {
-                  type: image.type,
-                  width: image.width,
-                  height: image.height
-              })
-            : transparentPlaceholder;
-
-        return (
-            <Image
-                alt={productName}
-                classes={{ root: classes.image }}
-                placeholder={transparentPlaceholder}
-                src={src}
-                fileSrc={image.url}
-                sizes={`${image.width}px`}
-            />
-        );
-    }, [hasImage, image, productName, classes.image]);
+        // Sizes
+        const imageProps = {
+            alt: productName,
+            classes: { root: classes.image },
+            sizes: `80px`
```

Comment:
Move this const to the top. Can merge with the other width setting ie `sizes: ${IMAGE_WIDTH}px`

Reply:
https://github.com/magento/pwa-studio/pull/1871/commits/3302c9b6f29dd756c1d1bb9c41a92c9d8c1c2073